cypress/README.md

```diff
@@ -0,0 +1,30 @@
+# Zaken frontend - Cypress integration
```

Comment:
Would be nice to mention the use of Cypress in the 'global' Readme and add a link to this Cypress-readme

Reply:
Good one! Do you agree with this separate readme file just for Cypress or should I include it in the global one?

Reply:
For readability I think it's good to have it separate, but with a clear link to the file from the global Readme.